episodes/1-introduction.Rmd

```diff
@@ -40,7 +40,7 @@ Deep learning (DL) is just one of many techniques collectively known as machine
 The image below shows some differences between artificial intelligence, machine learning and deep learning.
 
 
-![](fig/01_AI_ML_DL_differences.jpg){
+![](fig/01_AI_ML_DL_differences.png){
```

Comment:
The only thing I changed was renaming the definition for machine learning from 'Algorithms that learn from experience or data without human programming' to 'Algorithms that learn from experience or data without explicit instructions'